src/components/ConfigureGame/ConfigureGame.js

```diff
@@ -46,22 +46,14 @@ const ConfigureGame = () => {
             </div>
             <div className="player__actions">
               <Link to={`/player/edit/${player.id}`}>
-                <img
-                  className="player__icon"
-                  src={Pencil}
-                  alt={"Edit player"}
-                />
+                <img className="player__icon" src={Pencil} alt="Edit player" />
               </Link>
               {numberOfPlayers > MIN_PLAYERS && (
-                <Button
-                  styleReset
-                  onClick={() => removeInput(index)}
-                  aria-label="Remove player"
-                >
+                <Button styleReset onClick={() => removeInput(index)}>
```

Comment:
It's not removing an input anymore, so maybe `removePlayer` is a bit more descriptive?